### PR TITLE
Improve the look of README.md

### PR DESCRIPTION
**Pull request by turbaszek**
README is the first thing users will see so it's important to keep
it nice and structured. This commit adds more information about
Kibble components, structures the file and adds note about
contributing.

### CODE REVIEW

**Comment by turbaszek** [comment]
To see how the changes will look like please take a look: https://github.com/PolideaInternal/kibble/tree/improve-readme

File: README.md [hunk 1]
```diff
@@ -1,18 +1,33 @@
-![Logo](/ui/images/kibble-logo.png)
+<p align="center"><img src="/ui/images/kibble-logo.png" width="300"/></p>
 
-# Kibble UI and Database Service
+# Apache Kibble
 
-_Apache Kibble is a suite of tools for collecting, aggregating and
-visualizing activity in software projects._
+Apache Kibble is a tool to collect, aggregate and visualize data about open source projects
+maintained by open source projects. It consists of two components:
```
Comment (sharanf) [inline README.md]: Kibble will work for any software project not only the open source ones.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
+
+For installation steps see the [documentation](https://apache-kibble.readthedocs.io/en/latest/setup.html#installing-the-server).
 
-See the [documentation](https://apache-kibble.readthedocs.io/) for
-installation, setup and usage instructions.
+## Contributing
 
-This is one component of the Kibble Suite. Please also see the 
-[kibble scanners](https://github.com/apache/kibble-scanners) utility.
+We welcomes all contributions that improve the state of the Apache Kibble project. For contribution guidelines
```
Comment (sharanf) [inline README.md]: Minor typo - "We welcome all contributions..."

**Review by sharanf — APPROVED** [review]
Thanks for the contribution. This is a lot clearer for a new user wanting to try things out or get started. Some minor comments but nothing major.

File: README.md (hunk 2 above)
Comment (turbaszek) [inline README.md]:
```suggestion
We welcome all contributions that improve the state of the Apache Kibble project. For contribution guidelines
```

File: README.md (hunk 1 above)
Comment (turbaszek) [inline README.md]:
```suggestion
Apache Kibble is a tool to collect, aggregate and visualize data about any software project that uses commonly known tools. It consists of two components:
```

**Comment by turbaszek** [comment]
Hey @sharanf thanks a lot for the review! I applied your suggestions

**Pull request merged**